commit ce4565cb7523ab590e89d05a794d60c9c9dfc9c0
Author: Liviu Dudau <liviu@dudau.co.uk>
Date:   Wed Mar 1 12:26:28 2017 +0000

    ASoC: TLV320AIC23: Unquote NULL from control name
    
    commit a03faba972cb0f9b3a46d8054e674d5492e06c38 upstream.
    
    Without this I am getting the following messages at boot on my Trimslice:
       tlv320aic23-codec 2-001a: Control not supported for path LLINEIN -> [NULL] -> Line Input
       tlv320aic23-codec 2-001a: ASoC: no dapm match for LLINEIN --> NULL --> Line Input
       tlv320aic23-codec 2-001a: ASoC: Failed to add route LLINEIN -> NULL -> Line Input
       tlv320aic23-codec 2-001a: Control not supported for path RLINEIN -> [NULL] -> Line Input
       tlv320aic23-codec 2-001a: ASoC: no dapm match for RLINEIN --> NULL --> Line Input
       tlv320aic23-codec 2-001a: ASoC: Failed to add route RLINEIN -> NULL -> Line Input
       tlv320aic23-codec 2-001a: Control not supported for path MICIN -> [NULL] -> Mic Input
       tlv320aic23-codec 2-001a: ASoC: no dapm match for MICIN --> NULL --> Mic Input
       tlv320aic23-codec 2-001a: ASoC: Failed to add route MICIN -> NULL -> Mic Input
       tegra-snd-trimslice sound: tlv320aic23-hifi <-> 70002800.i2s mapping ok
    
    Signed-off-by: Liviu Dudau <liviu@dudau.co.uk>
    Signed-off-by: Mark Brown <broonie@kernel.org>

diff --git a/sound/soc/codecs/tlv320aic23.c b/sound/soc/codecs/tlv320aic23.c
index 410cae0f2060..628a8eeaab68 100644
--- a/sound/soc/codecs/tlv320aic23.c
+++ b/sound/soc/codecs/tlv320aic23.c
@@ -174,10 +174,9 @@ static const struct snd_soc_dapm_route tlv320aic23_intercon[] = {
 	{"ROUT", NULL, "Output Mixer"},
 
 	/* Inputs */
-	{"Line Input", "NULL", "LLINEIN"},
-	{"Line Input", "NULL", "RLINEIN"},
-
-	{"Mic Input", "NULL", "MICIN"},
+	{"Line Input", NULL, "LLINEIN"},
+	{"Line Input", NULL, "RLINEIN"},
+	{"Mic Input", NULL, "MICIN"},
 
 	/* input mux */
 	{"Capture Source", "Line", "Line Input"},
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1543_ce4565cb7523ab590e89d05a794d60c9c9dfc9c0.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1543_ce4565cb7523ab590e89d05a794d60c9c9dfc9c0.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
